Plugin/Framework/Search/Request/AbstractHandler.php

```diff
@@ -132,12 +132,15 @@ public function __construct(
 
     /**
      * @param array $requestData
+     * @param bool $doLog
      * @return void
      * @noinspection PhpRedundantCatchClauseInspection
      */
-    public function handle(array &$requestData)
+    public function handle(array &$requestData, $doLog)
```

Comment:
Why is this added here? Having a variable to control logging will add more confusion and make debugging harder.

Reply:
Since I am calling handle function no matter wether CMP is enabled or not I added logging argument to differentiate between where it actually logs something useful and where it just needs to be executed for initialising empty `CategoryMerchandisingResult` object